README.md

```diff
@@ -18,9 +18,11 @@ The repository is hosted in [GitHub](https://github.com/martincostello/website):
 
 This project is licensed under the [Apache 2.0](https://github.com/martincostello/website/blob/main/LICENSE) license.
 
-## Building and Testing
+## Building and Testing (Windows)
```

Comment:
The build script uses PowerShell 7, so is applicable to Windows, macOS and Linux.

```suggestion
## Building and Testing
```